rr_iarrc/launch/laplacian_line_detector.launch

```diff
@@ -3,15 +3,18 @@
         <!--Minimum Area to Keep-->
         <param name="perfect_lines_min_cut" type="int" value="50" />
         <!--Strength of Floodfill-->
-        <param name="Laplacian_threshold" type="int" value="40" />
-
+        <param name="Laplacian_threshold" type="int" value="60" />
+        <!--Strength of Adaptive Thresholding-->
         <param name="adaptive_mean_threshold" type="int" value="3" />
 
         <!--The larger the height the lower it will be on the screen-->
         <param name="blockSky_height" type="int" value="95" />
-        <param name="blockWheels_height" type="int" value="100" />
-        <param name="blockBumper_height" type="int" value="50" />
+        <param name="blockWheels_height" type="int" value="70" />
+        <param name="blockBumper_height" type="int" value="0" />
 
         <param name="subscription_node" type="string" value="/camera/image_color_rect"/>
     </node>
+
+    <include file="$(find rr_platform)/launch/sedani_state_publisher.launch"/>
```

Comment:
None of the line or cone detector launch files should launch the state publisher. That is the job of the gazebo or platform launch file, or of a testing launch file which includes sedani_state_publisher.launch for use with bag files. The same goes for the pointcloud converter launch file as well in the line below this one.